src/server/req_quejob.c

```diff
@@ -3550,12 +3549,17 @@ confirm_resv_locally(resc_resv *presv, struct batch_request *orig_preq)
 		return 1;
 	}
 
-	preq->rq_extend = strdup(PBS_RESV_CONFIRM_SUCCESS);
+	/* extend field format is "PBS_RESV_CONFIRM_SUCCESS:partition=<partition name>"
+	 * allocate enough memory to be able to support the format.
+	 */
+	extend_size = strlen(PBS_RESV_CONFIRM_SUCCESS) + strlen(partition_name) + 16;
```

Comment:
Could you explain what 16 is?  It's larger than strlen("partition=")

Reply:
oh right, it should be 12. I can change it if you want.

Reply:
I'd change it just because people will wonder why it is 4 characters more than it should be.